New version: SphereUDE v0.1.6

JuliaRegistrator opened the pull request:
- Registering package: SphereUDE
- Repository: https://github.com/ODINN-SciML/SphereUDE.jl
- Created by: @facusapienza21
- Version: v0.1.6
- Commit: 6692cc874c6d196898934175fe63208847b3448b
- Reviewed by: @facusapienza21
- Reference: https://github.com/ODINN-SciML/SphereUDE.jl/commit/6692cc874c6d196898934175fe63208847b3448b#commitcomment-149102241
- Description: A Julia package for regression in the sphere based on Universal Differential Equations
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd627f0f3be71a800311cba54fe7594ff6a716db4041cfc526429c870e6586c88aab40fcce7b27d26faf38899b5a4b8c85c6ea46a20b513583e1df4cf09eef8f35d3f4b34e889a3b439301affaa9e1f2449a4bb67f1f76a5a8e5d0bb38e682dc621240bb5d33496bde10c046554ce6d796dc9129159b399aa682bc50ce538a16028054b6903eb3f4df1cae455d2d2b3e02ba8ba0f6af5e0c2eb4a4a2ddee621db14f6259289e9fca7043da9aa6f9ca9c068 -->

facusapienza21 commented:
I am not sure what is the error with the automerge, since the other test have pass.

facusapienza21 commented:
[noblock]

JuliaRegistrator committed:
New version: SphereUDE v0.1.6

UUID: d7416ba7-148a-4110-b27d-9087fcebab2d
Repo: https://github.com/ODINN-SciML/SphereUDE.jl.git
Tree: 6416af0a06607801320a484cb24a5664964158d5

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
